### PR TITLE
YLP-1753 Fix memory leak on useState

### PR DESCRIPTION
**Pull request by Xwdrblue**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,9 @@
 Blip is the web front end for YourLoops system.
 It is based on Tidepool Blip 1.27.
 
-## 3.0.2-beta6 - 2022-10-21
+## 3.0.2-beta8 - 2022-10-25
 ### Added
+- YLP-1753 Fix memory leak on useState
```
Comment (fatb38) [inline CHANGELOG.md:7]: must be sorted

**Review by fatb38 — DISMISSED** [review]

**Review by TimCanu — APPROVED** [review]
Looking good. Again, do not merge without the yourloops PR :slightly_smiling_face:

**Comment by Xwdrblue** [comment]
> Looking good. Again, do not merge without the yourloops PR slightly_smiling_face

**Comment by Xwdrblue** [comment]
The beta 8 turned with the beta 8 of yourloops

**Pull request merged**